agent/agent_test.go

```diff
@@ -3278,10 +3278,11 @@ func TestAgent_purgeCheckState(t *testing.T) {
 
 func TestAgent_GetCoordinate(t *testing.T) {
 	t.Parallel()
-	check := func(server bool) {
-		a := NewTestAgent(t, `
-			server = true
-		`)
+	check := func(t *testing.T, server bool) {
+		a := NewTestAgent(t, fmt.Sprintf(`
+			server = %v
+			bootstrap = %v
```

Comment:
The test was only actually checking `server`. I had to set bootstrap=false to make it work for clients.

I'm not exactly sure what this test is for. Seems like a regression test for a very specific bug.

Reply:
Good call. I replaced the old regression test with a happy-path test that checks the return value. From what i could tell, the coordinate value is predictable in this case because there is only a single server. Does that sound right?